docs/Authentication.md

```diff
@@ -315,3 +315,72 @@ export default withRouter(MyPage);
 {% endraw %}
 
 The `<Authenticated>` component calls the `authClient` function with `AUTH_CHECK` and `authParams`. If the response is a fulfilled promise, the child component is rendered. If the response is a rejected promise, `<Authenticated>` redirects to the login form. Upon successful login, the user is redirected to the initial location (that's why it's necessary to get the location from the router).
+
+
+## Redirect after logout
+
```

Comment:
please add a list item in the left menu for that section.

Reply:
Where should I add this? `_layouts/default.html` ?

Reply:
yes, exactly